base/src/Data/Macaw/Analysis/RegisterUse.hs

```diff
@@ -678,6 +679,8 @@ processApp aid app = do
       setAssignVal aid (FrameExpr (o+fromInteger c))
     BVSub _ (FrameExpr o) (IVCValue (BVCValue _ c)) ->
       setAssignVal aid (FrameExpr (o-fromInteger c))
+    BVAnd _ (FrameExpr o) (IVCValue (BVCValue _ c)) ->
```

Comment:
Does it also make sense to add a `BVAnd _ (IVCValue (BVCValue _ c)) (FrameExpr o)` case for symmetry with the two `BVAdd` cases above? (I'm not sure if this case would ever occur in practice, but I did find myself wondering about its omission here.)

Reply:
I agree that in practice this version is the most likely to be generated by Macaw, but it doesn't hurt to add, so I will.